RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerComponentViewModel.cs

```diff
@@ -8,13 +8,12 @@
 using Rubberduck.VBEditor;
 using resx = Rubberduck.UI.CodeExplorer.CodeExplorer;
 using Rubberduck.Parsing.Annotations;
-using Rubberduck.Parsing.Grammar;
 
 namespace Rubberduck.Navigation.CodeExplorer
 {
     public class CodeExplorerComponentViewModel : CodeExplorerItemViewModel
     {
-        private readonly Declaration _declaration;
+        public Declaration Declaration { get; private set; }
```

Comment:
This contrasts with the rest of the project, where get-only immutable properties are implemented like this:

    private readonly Declaration _declaration;
    public Declaration Declaration { get { return _declaration; } }

The difference is that the `_declaration` backing field is marked `readonly`, which means *even the private internals of the class* cannot tamper with that reference; on the other hand a `private set` property doesn't clearly mean "immutable", the private implementation of the class is free to reassign that reference in the lifetime of the object.

In C# 6 you can have truly readonly properties. Before that, you had to use a `private readonly` field.